### DISCUSSION TITLE
New issue templates

### DISCUSSION THREAD

**Discussion by link3280** [apache/kyuubi#1824] [General]
Currently, we have mainly two types of issues: bug reports and feature requests, and some efforts (e.g. refactoring for code style/performance) may not fall into these categories. So I propose to add an `improvements` template for these efforts.

**Comment by yaooqinn** [comment]
I am OK with this cc @yanghua

**Comment by yanghua** [comment]
+1, sorry for the late reply, I opened "All Activity" in the Github "Watch". So I was inundated by so many messages.
